docs/src/developing/clients/jsonrpc-api.md

```diff
@@ -2668,9 +2683,10 @@ Submits a signed transaction to the cluster for processing.
 Before submitting, the following preflight checks are performed:
 
 1. The transaction signatures are verified
-2. The transaction is simulated against the latest max confirmed bank
-   and on failure an error will be returned. Preflight checks may be disabled if
-   desired.
+2. The transaction is simulated against the bank slot specified by the preflight
+   commitment, and on failure an error will be returned. Preflight checks may be
+   disabled if desired. It is recommended to specify the same commitment and
+   preflight commitment to avoid confusing behavior.
```

Comment:
We don't have checks for this, which makes me think there is a use case for different levels of commitment on simulation and processing.  Perhaps this gets updated as well in web3 and rpc client?

Reply:
More that in the past, it wasn't possible to tune the commitment of the simulation. I haven't thought of a strong use case for simulating and processing at different commitments.
There is client probably work we can do to make this smoother/less confusing.